### ISSUE TITLE
Current master is broken against stable vscode due to disagreeing tagSupport coming in lsp 3.15

### ISSUE THREAD

**Issue by udoprog** [rust-analyzer/rust-analyzer#2651]
The latest vscode (1.41.1) contains a version of `vscode-languageserver-node` which uses the boolean `true` for `tagSupport`.

This leads to the following error with the current `master` rust-analyzer:
```
Error: Error("invalid type: boolean `true`, expected struct TagSupport", line: 0, column: 0)
```

This was fixed in the next version of vscode-languageserver-node, but I don't know when it will be bumped. Probably not until the next version of lsp is stabilized (3.15):
https://github.com/microsoft/vscode-languageserver-node/commit/1b637ef62a120f5a263e994bb21065230c959829

**Comment by shepmaster** [comment]
I believe that this update broke emacs support as I get an error like `` missing field `tagSupport` ``

**Comment by udoprog** [comment]
@shepmaster should've been fixed in #2659, sorry for the noise!

**Comment by shepmaster** [comment]
Thank you; no real noise for me, just giving rust-analyzer a shot.
